docs/pages/inboxes/history-sync.mdx

```diff
@@ -33,26 +33,34 @@ You can choose to [run your own server](https://github.com/xmtp/xmtp-message-his
 When your app initializes an XMTP client and a `historySyncUrl` client option is present, history sync automatically triggers an initial sync request and creates an encrypted payload.
 
 <div>
-    <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/history-sync-step1.png" width="600px" />
+    <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/history-sync-1-state.png" width="600px" />
 </div>
 
-History sync then uploads the payload, sends a sync reply, and pulls all conversation state history into the new app installation, merging it with the existing app installations in the sync group.
+History sync then uploads the payload, sends a sync reply, and pulls all conversation state history into the new app installation, merging it with the existing app installations in the installation state group.
 
 <div>
-    <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/history-sync-step2.png" width="600px" />
+    <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/history-sync-2-state.png" width="600px" />
 </div>
 
-Ongoing updates to history are streamed automatically. Updates, whether for user consent preferences or messages, are sent across the sync group, ensuring all app installations have up-to-date information.
+Ongoing updates to history are streamed automatically. Updates, whether for user consent preferences or messages, are sent across the installation state group, ensuring all app installations have up-to-date information.
 
 History syncs are accomplished using these components:
 
-- Sync group
+- Installation state group
 - Sync worker
 - History server
 
-### Sync group
+### Installation state group
```

Comment:
@nplasterer thank you for your comments on this PR!

How about "installation state group"?